packages/pie-inline-choice/src/inline-choice.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import PropTypes from 'prop-types';
+import { withStyles } from 'material-ui/styles';
+import Input, { InputLabel } from 'material-ui/Input';
+import { MenuItem } from 'material-ui/Menu';
+import { FormControl, FormHelperText } from 'material-ui/Form';
+import Select from 'material-ui/Select';
+
+const styles = theme => ({
+  container: {
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+  formControl: {
+    margin: theme.spacing.unit,
+    minWidth: 120,
+    maxWidth: 300
+  },
+  selectEmpty: {
+    marginTop: theme.spacing.unit * 2,
+  }
+});
+
+
+class InlineChoice extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
do we need `state`?

Reply:
Yes I am setting the selected value in state on change